R/designSampleSize.R

```diff
@@ -257,39 +260,104 @@ designSampleSizePlots = function(data) {
         stop ("Invalid input")
     }
     
-    text.size = 1.2	
-    axis.size = 1.3	
-    lab.size = 1.7
-    
+    if(isPlotly) {
+        axis.size <- 10
+        lab.size <- 12 
+        text.size <- 12
```

Comment:
To be consistent, could these be `=` signs instead of `<-`.

Reply:
Done! Thanks